packages/client/src/components/Outcome/NewCheckerModal.tsx

```diff
@@ -42,115 +32,60 @@ const NewCheckerModal: FunctionComponent = () => {
       action: actions.OPEN_MODAL,
       name: eventNames.OPEN_MODAL_DO_ANOTHER_CHECK,
     });
-
-    setError(false);
   };
 
   const handleConfirmButton = () => {
-    const hasError = saveAddress === null;
+    setFinished(!!checkerSlug);
+
+    matomoTrackEvent({
+      action: actions.START_ANOTHER_CHECK,
+      name: eventNames.DO_ANOTHER_CHECK,
+    });
 
-    setError(hasError);
+    // Set the new topic data with the correct `type` and potentially the `address`
+    const newTopicData = {
+      ...defaultTopicSession,
+      // Optionally save address
+      address: topicData.address,
+      // Update timesCheckerLoaded + 1 or reset to zero if going to a new checker
+      timesCheckerLoaded:
+        checkerSlug === slug ? topicData.timesCheckerLoaded + 1 : 0,
+      type: checkerSlug,
+    };
+
+    if (checkerSlug === slug) {
+      // Only change the topicData for the current topic
+      setTopicData(newTopicData);
 
-    if (hasError) {
       matomoTrackEvent({
-        action: actions.START_ANOTHER_CHECK,
-        name: `${eventNames.DO_ANOTHER_CHECK} - ${eventNames.NO_CHOICE_HAS_BEEN_MADE}`,
+        action: actions.CLICK_INTERNAL_NAVIGATION,
+        name: sections.INTRO,
       });
     } else {
-      setFinished(!!checkerSlug);
-
-      const doSaveAddress = saveAddress === true;
-
-      matomoTrackEvent({
-        action: actions.START_ANOTHER_CHECK,
-        name: `${eventNames.DO_ANOTHER_CHECK} - ${
-          doSaveAddress
-            ? eventNames.WITH_THE_SAME_ADDRESS
-            : eventNames.WITHOUT_THE_SAME_ADDRESS
-        }`,
+      // Change the topicData for another topic
+      setSession({
+        [checkerSlug]: newTopicData,
       });
-
-      // Set the new topic data with the correct `type` and potentially the `address`
-      const newTopicData = {
-        ...defaultTopicSession,
-        // Optionally save address
-        address: doSaveAddress ? topicData.address : null,
-        // Update timesCheckerLoaded + 1 or reset to zero if going to a new checker
-        timesCheckerLoaded:
-          checkerSlug === slug ? topicData.timesCheckerLoaded + 1 : 0,
-        type: checkerSlug,
-      };
-
-      if (checkerSlug === slug) {
-        // Only change the topicData for the current topic
-        setTopicData(newTopicData);
-
-        matomoTrackEvent({
-          action: actions.ACTIVE_STEP,
-          name: doSaveAddress ? sections.QUESTIONS : sections.LOCATION_INPUT,
-        });
-      } else {
-        // Change the topicData for another topic
-        setSession({
-          [checkerSlug]: newTopicData,
-        });
-      }
-
-      // Clear checker and go to the new topic slug
-      setChecker(undefined);
-      history.push(geturl(routes.checker, { slug: checkerSlug }));
     }
+
+    // Clear checker and go to the new topic slug
+    setChecker(undefined);
+    history.push(geturl(routes.intro, { slug: checkerSlug }));
```

Comment:
Going to the intro page is only necessary when the checker is different from the current checker.

Current checkers should go either to the Location or Question Section. I will discuss today with UX and get back to you.